script.js

```diff
@@ -1,33 +1,57 @@
 // 1. Create An Array Of Students.
 let studentsOne = [
-  { name: "Hammad", age: 17 },
-  { name: "Abdul Rafay", age: 11 },
-  { name: "Huzaifa", age: 8 },
-  { name: "Laraib", age: 16 },
+  { name: "Hammad", age: 17, class: "10th" },
+  { name: "Abdul Rafay", age: 11, class: "3rd" },
+  { name: "Huzaifa", age: 8, class: "2nd" },
+  { name: "Laraib", age: 16, class: "9th" },
 ];
 
 let teachersOne = [
-  { name: "Sir Bilal", age: 27, salary: 200 },
-  { name: "Sir Areeb", age: 21 },
-  { name: "Sir Nabeel", age: 28 },
-  { name: "Sir Habeeb", age: 26 },
+  { name: "Sir Bilal", age: 27, subjects: "English" },
+  { name: "Sir Areeb", age: 21, subjects: "Chemistry" },
+  { name: "Sir Nabeel", age: 28, subjects: "Physics" },
+  { name: "Sir Habeeb", age: 26, subjects: "Urdu" },
 ];
 
 // 2.Create a function named renderTable.
 let rederTable = function (items) {
   // 3. Create table dynamically .
   let container = document.querySelector("#table");

```

Comment:
Now we should take 2 arguments  `items` and 'elementId'

So we can decide where we will append the table.

```suggestion
let rederTable = function (items, elementId) {
  // 3. Create table dynamically .
  let container = document.querySelector(elementId);

```